templates/zerver/team.html

```diff
@@ -141,6 +157,10 @@ <h1 id="our-amazing-community">Our amazing community</h1>
                     </p>
                 </script>
 
+                <script type="text/template" id="contributor-count-template">
+                    <p class="contributor-count">Total contributors: <%= contributor_count %></p>
```

Comment:
Will this always be a grand total, or just the total for a given (sub)project?  It might be good to make that clear, e.g. "Total server contributors" or similar when you're in those tabs.

Reply:
Or just "353 server contributors" might feel like cleaner language?